tests/FactoryTest.php

```diff
@@ -2,14 +2,15 @@
 
 namespace Christophrumpel\LaravelFactoriesReloaded\Tests;
 
-use ExampleApp\Models\Group;
-use ExampleApp\Models\Ingredient;
-use ExampleApp\Models\Recipe;
 use ExampleAppTests\Factories\GroupFactory;
 use ExampleAppTests\Factories\GroupFactoryUsingFaker;
+use ExampleAppTests\Factories\IngredientFactoryUsingClosure;
 use ExampleAppTests\Factories\RecipeFactory;
 use ExampleAppTests\Factories\RecipeFactoryUsingFactoryForRelationship;
 use ExampleAppTests\Factories\RecipeFactoryUsingLaravelFactoryForRelationship;
+use ExampleApp\Models\Group;
+use ExampleApp\Models\Ingredient;
+use ExampleApp\Models\Recipe;
```

Comment:
I think my IDE just auto-alphabetized these.

Reply:
yeah, im not opinionated on this either tbh. Whenever i auto-import something, it just re-alphabetizes the list. I can manually put it back if you like.

Reply:
It's ok, just leave it 👍